app/routes/authRoutes.js

```diff
@@ -10,6 +10,7 @@ const router = express.Router();
 router.get('/api/v1/captcha', asyncHandler(AuthController.getCaptcha));
 router.post('/api/v1/auth/login', validateParams(signinValidatorSchema), asyncHandler(AuthController.signin));
 router.delete('/api/v1/auth/logout', validateParams(exitValidatorSchema), authMiddleware, asyncHandler(AuthController.exit));
-router.patch("/api/v1/auth/resetPassword", asyncHandler(AuthController.resetPassword))
+router.patch('/api/v1/auth/resetPassword', asyncHandler(AuthController.resetPassword));
```

Comment:
Bad naming practice of the API and it doesn't follow the RESTful API design pattern.

1. It's better using PATCH /api/v1/auth/login to update the login information, password is considered as part of the login info as weel.
2. POST /api/v1/auth/activation instead of /api/v1/auth/sendActivateEmail

There's a simple rule here, do not use verb + noun in the URI, since the http methods have the meaning of actions. For instance, POST stands for create, GET stands for get, PATCH stands for update, etc.